apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -336,34 +364,26 @@ protected String getConfigText(StreamConfiguration config) {
 
     @Override
     protected String shouldTestBeSkipped() {
-        String why = "";
+        String why = super.shouldTestBeSkipped();
```

Comment:
It feels a bit odd to me why this returns a String instead of a bool. Consider changing the name of this function

Reply:
I changed it to whyShould...